src/lua/lua_task.c

```diff
@@ -2811,6 +2828,43 @@ lua_task_get_symbol (lua_State *L)
 }
 
 static gint
+lua_task_get_symbols_all (lua_State *L)
+{
+	struct rspamd_task *task = lua_check_task (L, 1);
+	struct rspamd_metric *metric;
+	struct rspamd_metric_result *mres;
+	GHashTableIter it;
+	gpointer k;
+	gboolean found = FALSE;
+	gint i = 1;
+
+	if (task) {
+		metric = task->cfg->default_metric;
+		mres = g_hash_table_lookup (task->results, DEFAULT_METRIC);
+		if (mres) {
+			found = TRUE;
+			lua_newtable (L);
```

Comment:
It is better to use lua_createtable here since we know the upper bound of the table.